crates/bevy_pbr/src/light_probe/environment_map.wgsl

```diff
@@ -206,9 +205,8 @@ fn environment_map_light(
     var out: EnvironmentMapLight;
 
     let radiances = compute_radiances(
-        input,
+        (*input).layers[LAYER_BASE],
```

Comment:
Did you test the performance here? We used to use by-value struct passing and it had severe impacts on our performance that @Elabajaba noticed (that's why I changed it to `ptr` in the first place).

Reply:
I didn't, but we use all 5 fields of the struct anyways, so I don't see what the difference could be. I'll try and profile.

Reply:
Tested perf, no real difference.